SlicerRadiomics/SlicerRadiomics.py

```diff
@@ -278,8 +295,15 @@ def onApplyButton(self):
     kwargs['verbose'] = self.verboseCheckBox.checked
     kwargs['label'] = int(self.labelSliderWidget.value)
```

Comment:
@fedorov, if you disable the slicer widget, it might be a good idea to also disable this line of code, which sets the label value. It won't produce bugs in its current form, as the widget is still created and has value 1, which is the default value for label.